---
source: src/validation/tests/array_validation_test.rs
assertion_line: 79
expression: res
---
SyntaxError { message: "Array initializer must be an initializer list!", range: [SourceRange { range: 314..318 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Expected identifier or '('", range: [SourceRange { range: 325..326 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Array initializer must be an initializer list!", range: [SourceRange { range: 449..453 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Expected identifier or '('", range: [SourceRange { range: 460..461 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Array initializer must be an initializer list!", range: [SourceRange { range: 497..500 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Expected identifier or '('", range: [SourceRange { range: 507..508 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Array initializer must be an initializer list!", range: [SourceRange { range: 603..607 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Expected identifier or '('", range: [SourceRange { range: 614..615 }], err_no: arr__invalid_array_assignment }
SyntaxError { message: "Could not resolve reference to var1", range: [SourceRange { range: 592..596 }], err_no: reference__unresolved }
SyntaxError { message: "Could not resolve reference to var2", range: [SourceRange { range: 603..607 }], err_no: reference__unresolved }
SyntaxError { message: "Could not resolve reference to var1", range: [SourceRange { range: 647..651 }], err_no: reference__unresolved }
SyntaxError { message: "Could not resolve reference to var2", range: [SourceRange { range: 658..662 }], err_no: reference__unresolved }

